scripts/comparison-test.sh

```diff
@@ -6,14 +6,7 @@ set -eo pipefail
 SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 cd "$SCRIPT_DIR" || exit 1
 
-DATA="zq-sample-data"
-DATA_REPO="https://github.com/brimsec/zq-sample-data.git"
-if [ -d "$DATA" ]; then
-  (cd "$DATA"&& git pull) || exit 1
```

Comment:
If you do retain this, nit: use spacing, as in
```bash
  (cd "$DATA" && git pull) || exit 1
```

Reply:
Since I'm opting to not retain, this nit should not apply.